src/diagram/models/mathjs-utils.test.ts

```diff
@@ -1,6 +1,26 @@
-import { replaceInputNames } from "./mathjs-utils";
+import {getMathUnit, replaceInputNames} from "./mathjs-utils";
 
 describe("mathjs-utils", () => {
+  describe("getMathUnit", () => {
+    it("get standard unit", () => {
+      const result = getMathUnit(0,"mm");
+      expect(result?.toString()).toBe("0 mm");
+    });
+    it("get custom unit", () => {
+      const result = getMathUnit(0, "cat");
+      expect(result?.toString()).toBe("0 cat");
+    });
+    it("plural of existing custom unit is same", () => {
+      const singU = getMathUnit(0, "mouse");
+      const plurU = getMathUnit(0, "mice");
+      expect(singU && plurU?.equals(singU)).toBe(true);
+    });
+    it("plural of existing custom unit is same 2", () => {
+      const singU = getMathUnit(0, "halves");
+      const plurU = getMathUnit(0, "half");
```

Comment:
```suggestion
      const singU = getMathUnit(0, "half");
      const plurU = getMathUnit(0, "halves");
```
Aren't these reversed?

Reply:
Thanks, I made the test to verify you can create the unit from the plural before the singular exists, but I should have fixed the naming.